ddtrace/utils/formats.py

```diff
@@ -13,7 +13,8 @@ def get_env(integration, variable, default=None):
           arguments
         * return `default` otherwise
     """
-    key = '{}_{}'.format(integration, variable).upper()
+    key = '{}_{}'.format(integration, variable) if integration else variable
```

Comment:
this is tricky, since I'd assume I can do `get_env('var')` to fetch `DD_VAR`.

We can get around this with something like:

```python
parts = [p.strip().upper() for p in (integration, variable) if p.strip()[
if not parts:
    # Eh ?
    # Raising exceptions usually isn't the best of ideas, maybe we just always return back the default in this case?
    raise Exception()

legacy_env = '_'.join(['DATADOG'] + parts)
env = '_'.join(['DD'] + parts)
```


Actually... with this change we can just do:

```python
def get_env(*args, default=None):
    args = [a.strip().upper() for a in args if a.strip()]
    if not args:
        return None
    
    legacy_env = '_'.join(['DATADOG'] + parts)
    env = '_'.join(['DD'] + parts)
```

The idea being that it is just positional, so you can do:

```python
get_env('service_name')  # DD_SERVICE_NAME
get_env('redis', 'service_name')  # DD_REDIS_SERVICE_NAME
get_env('redis', 'service', 'name')  # DD_REDIS_SERVICE_NAME
get_env('service_name', default='service-name')
```

Reply:
Good ideas. I'll open a different PR with the changes to `get_env` so we can focus this discussion there. I've reverted the changes in this PR.

Reply:
:+1: